cmd/ocsp-responder/main.go

```diff
@@ -141,8 +141,15 @@ func main() {
 
 		config := c.OCSPResponder
 		var source cfocsp.Source
-		url, err := url.Parse(config.Source)
-		cmd.FailOnError(err, fmt.Sprintf("Source was not a URL: %s", config.Source))
+
+		// DBConfig takes precedence over Source, if present.
+		dbConnect, err := config.DBConfig.URL()
+		cmd.FailOnError(err, "Reading DB config")
+		if dbConnect == "" {
+			dbConnect = config.Source
+			cmd.FailOnError(err, fmt.Sprintf("Source was not a URL: %s", config.Source))
```

Comment:
This line is wrong.  You already fail on this err three lines above.  Are you missing a call to `url.Parse()`?